src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts

```diff
@@ -415,6 +422,10 @@ export class BackLayerWebView<T extends ICommonCellInfo> extends Disposable {
 						color: var(--vscode-foreground);
 					}
 
+					#container > div > div.preview.selected {
+						background: var(--vscode-notebook-selectedCellBackground) !important;
```

Comment:
The catch with using `!important` is we may want to support decorations for markdown cells in the webview and the decoration background won't win. https://github.com/microsoft/vscode/issues/118000

Reply:
Thanks for the recommendation! I've update the PR to also support decorations